integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -52,12 +53,12 @@ def transform_provider_id(self, content_metadata_item):  # pylint: disable=unuse
         """
         return self.enterprise_configuration.provider_id
 
-    def transform_status(self, content_metadata_item):
+    def transform_for_action_status(self, content_metadata_item, action):
         """
         Return the status of the content item.
         """
         # lets not overwrite something we've already tried to set INACTIVE
-        if content_metadata_item.get('status') == 'INACTIVE':
+        if content_metadata_item.get('status') == 'INACTIVE' or action == IC_DELETE_ACTION:
```

Comment:
i believe you only need to branch on `action == IC_DELETE_ACTION` - i'd remove the `content_metadata_item.get('status') == 'INACTIVE'` part. this was added to deal with the old way of doing things where we were setting the status in the transmit and we didnt want the transformer to stomp on that change.